src/main/java/no/nav/fo/veilarbregistrering/profilering/ProfileringService.java

```diff
@@ -0,0 +1,31 @@
+package no.nav.fo.veilarbregistrering.profilering;
+
+import no.nav.fo.veilarbregistrering.arbeidsforhold.ArbeidsforholdGateway;
+import no.nav.fo.veilarbregistrering.arbeidsforhold.FlereArbeidsforhold;
+import no.nav.fo.veilarbregistrering.besvarelse.Besvarelse;
+import no.nav.fo.veilarbregistrering.bruker.Foedselsnummer;
+
+import java.time.LocalDate;
+
+public class ProfileringService {
+
+    private final ArbeidsforholdGateway arbeidsforholdGateway;
+
+    public ProfileringService(ArbeidsforholdGateway arbeidsforholdGateway) {
+        this.arbeidsforholdGateway = arbeidsforholdGateway;
+    }
+
+    public Profilering profilerBruker(
+            int alder, //TODO: Fjerne alder fra signatur - utledes av fnr på innsiden, eller hentes fra PDL. Det første krever litt innsats med testdata.
```

Comment:
Som det står i TODO, kan egentlig alder nå fjernes som følge av at vi tar inn fødselsnummer. Men det krever at vi lager et fødselsnummer for test som har 3 gitte aldre.

Vi bør også ta i. bruk PDL og henting av alder derfra i stedet.

Reply:
Det vil være et steg nærmere fra å gjøre profileringen til noe som kan leve mer på egenhånd